app/react/App/scss/elements/_map.scss

```diff
@@ -548,8 +548,27 @@ a.mapboxgl-ctrl-logo {
   @include tooltip('right');
 }
 
+@media print {
+  /* stylelint-disable */
```

Comment:
I don't think this disable is needed? I removed the comments and my linter is not complaining.

Reply:
Sure, I'll remove it.